suites/pacific/rados/test-location-stretch-mode.yaml

```diff
@@ -0,0 +1,80 @@
+# cluster-conf file: conf/pacific/rados/stretch-mode-host-location-attrs.yaml
+
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap with apply-spec
+      desc: Apply spec in Bootstrap with host location attributes
+      module: test_bootstrap.py
+      polarion-id:
```

Comment:
Sorry... I believe this was missed out in my earlier review... could we add a Polarion ID, please

Reply:
Updated.